hw/ip/keymgr/data/keymgr.hjson

```diff
@@ -13,14 +13,15 @@
     { name: "op_done",   desc: "Operation complete" },
   ],
   alert_list: [
-    { name: "fatal_fault_err",
-      desc: "Alert for key manager faults.  These errors cannot be caused by software",
-    },
     { name: "recov_operation_err",
       desc: '''
         Alert for key manager operation errors.  These errors could have been caused by
         software'''
-    }
+    },
+    { name: "fatal_fault_err",
+      desc: "Alert for key manager faults.  These errors cannot be caused by software",
+    },
+
```

Comment:
Since the order of these were moved around, you will need to re-run `util/make_new_dif.py --mode=regen --only=autogen'` to update the autogen'd DIFs. Also make sure you have the right version of clang-format inplace before running the DIF autogen again by running the commands [here](https://github.com/lowRISC/opentitan/issues/8604#issuecomment-943607668).

Reply:
yep! it's just that my local repo had moved on to another ongoing PR, so I'll come back and fix this when I'm done with that.

Reply:
Ah got it. Sorry about that!

Reply:
fixed!